Makefile

```diff
@@ -152,6 +152,8 @@ pull/%: DARGS?=
 pull/%: ## pull a jupyter image
 	docker pull $(DARGS) $(OWNER)/$(notdir $@)
 
+pull-all: $(foreach I,$(ALL_IMAGES),pull/$(I) ) ## pull all images
```

Comment:
This is useful if I want to pull all the latest images (to make sure local testing works just like in master).